interface/forms/phq9/report.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * PHQ-9 report.php
+ * display a form's values in the encounter summary page
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once(dirname(__FILE__) . '/../../../library/api.inc');
```

Comment:
another change that will be happening today :) (adding the .php to end of api.inc)
```
require_once(dirname(__FILE__) . '/../../../library/api.inc.php');
```